pkg/scheduler/eventhandlers.go

```diff
@@ -377,6 +397,20 @@ func addAllEventHandlers(
 		},
 	)
 
+	informerFactory.Storage().V1().CSIDrivers().Informer().AddEventHandler(
+		cache.ResourceEventHandlerFuncs{
+			AddFunc:    sched.onCSIDriverAdd,
+			UpdateFunc: sched.onCSIDriverUpdate,
+		},
+	)
```

Comment:
#101394 attempted to make the logic of event handler registration dynamic.

Let's wait for it to be merged, and then the code here can be simplified.